pyproject.toml

```diff
@@ -1,18 +1,32 @@
+[build-system]
+requires = [
+    "hatchling",
+    "hatch-vcs",
```

Comment:
I don't think we want to use a VCS version if we end up publishing to PyPi; I don't really understand the reason for wanting it.

Reply:
It's mainly a maintenance thing so you don't have to remember to change static versioning.  setuptools-scm is pretty robust and does not need to write anything to a version file (unless you want it to).  BUT, again I can lose that bit if you want it the way it was.